include/zephyr/bluetooth/classic/hfp_hf.h

```diff
@@ -168,16 +427,492 @@ struct bt_hfp_hf_cb {
  */
 int bt_hfp_hf_register(struct bt_hfp_hf_cb *cb);
 
-/** @brief Handsfree client Send AT
+/** @brief Create the hfp hf session
+ *
+ *  Create the hfp hf session
+ *
+ *  @param conn ACL connection object.
+ *  @param hf Created HFP HF object.
+ *  @param channel Peer rfcomm channel to be connected.
+ *
+ *  @return 0 in case of success or negative value in case of error.
+ */
+int bt_hfp_hf_connect(struct bt_conn *conn, struct bt_hfp_hf **hf, uint8_t channel);
+
+/** @brief Disconnect the hfp hf session
+ *
+ *  Disconnect the hfp hf session
+ *
+ *  @param hf HFP HF object.
+ *
+ *  @return 0 in case of success or negative value in case of error.
+ */
+int bt_hfp_hf_disconnect(struct bt_hfp_hf *hf);
```

Comment:
Is it valid to still use the `hf` pointer after calling this function, or is it "freed"? (this should all be documented)

Reply:
Updated